libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -1465,12 +1602,26 @@ protected void onAppBackgrounded() {
         ((ScreenLockManager) getScreenLockManager()).onAppBackgrounded();
         ((BiometricAuthenticationManager) getBiometricAuthenticationManager())
                 .onAppBackgrounded();
+
+        // Hide the Salesforce Mobile SDK "Show Developer Support" notification
+        if (authenticatedActivityForDeveloperSupport != null) {
```

Comment:
I piggy-backed on this existing application lifecycle event to hide the notification when the app is delivered to the background.